docs/sources/variables/verify-option-get-from-url.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Verify option get from url"
+weight = 500
++++
+
+# Verify option
```

Comment:
Where do you turn this on? If it is in the config file, then I recommend updating configuration.md.

Also, I don't understand what this option is or why I might want to use it. Examples or a simple use case would be helpful if this documentation stands.

Reply:
When you find out, please update the title of this PR so it makes sense in context of RNs.